### DISCUSSION TITLE
Is there a manual mesh available?

### DISCUSSION THREAD

**Discussion by Miq1** [mriscoc/Ender3V2S1#11] [General]
Coming from the jyers firmware that has UBL without a probe integrated, I would like to know if that was possible with your firmware as well?
And if so: is there a beginner's guide to building the firmware from the sources? I got MSVC and Platformio, if that helps.

**Comment by Cal-LF** [comment]
@Miq1  DL the zip in the green Code button. In the config folder you will find config files. I think UBL is not used in this firmware.
